dspy/teleprompt/gepa/gepa.py

```diff
@@ -211,6 +211,13 @@ def metric(
             a strong reflection model. Consider using `dspy.LM(model='gpt-5', temperature=1.0, max_tokens=32000)` 
             for optimal performance.
         skip_perfect_score: Whether to skip examples with perfect scores during reflection. Default is True.
+        instruction_proposer: Optional custom instruction proposer implementing GEPA's ProposalFn protocol. 
+            If provided, GEPA will use this custom proposer instead of its default instruction proposal 
```

Comment:
Can you include some link to the `default instruction proposal` so that people wanting to implement their own logic have a reference point? Also some kind of link to `MultiModalInstructionProposer() from dspy.teleprompt.gepa.instruction_proposal`.

Reply:
I'm not sure how to add links here. I've just added this:

```
Use `MultiModalInstructionProposer()` from `dspy.teleprompt.gepa.instruction_proposal` 
for handling visual content. If None (default), GEPA uses its built-in text-optimized 
proposer (see `gepa.strategies.instruction_proposal.InstructionProposalSignature` 
for reference implementation).
```

Is this what you were proposing? or is there a way to add hrefs here that I might be familiar with? 🤔

Reply:
I think this is markdown so \[]() should work. I think you can feel free to ignore this as nit for now, otherwise, just include a markdown link to the source location. Won't hold back merge for this!